tests/layer_tests/tensorflow_tests/test_tf_BroadcastArgs.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (C) 2018-2023 Intel Corporation
```

Comment:
Not sure it is correct copyright, I may expect (c) 2023 because it is a new file

Reply:
Absolutely agree. Better to concentrate on technical things:)